androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/PlayerControls.kt

```diff
@@ -49,8 +57,10 @@ fun PlayerControls(
     onSettingsToggle: () -> Unit,
     onSeekBarValueFinished: (Long) -> Unit,
     onSeekBarValueChange: (Long) -> Unit,
+    setDeviceVolume: (Int) -> Unit,
     customization: ControlsCustomization
 ) {
+    val shouldShowContent = !isBrightnessSliderDragged && !isVolumeSliderDragged
```

Comment:
Instead of passing `isBrightnessSliderDragged` and `isVolumeSliderDragged`, just pass `shouldShowContent`

Reply:
i need the value of `isBrightnessSliderDragged` separated to control the shadow of playerControls

Reply:
i need the value of `isBrightnessSliderDragged` separated to control the shadow of PlayerControls when the user is dragging the brightness slider. take a look in `PlayerControlsScaffold`

Reply:
Got it. I couldn't see this usage on PR, so I thought that wasn't been used anymore.